README_GCP.md

```diff
@@ -41,26 +41,18 @@ gcloud iam service-accounts add-iam-policy-binding opentelemetry-demo@${GCLOUD_P
     --member "serviceAccount:${GCLOUD_PROJECT}.svc.id.goog[otel-demo/opentelemetry-demo-otelcol]"
 ```
 
-Update [`gcp-config-valus.yml`](gcp-config-values.yml) to annotate the
-Kubernetes service account with your project:
+### Deploying the Helmfile
```

Comment:
Could you add a link to the Helmfile prerequisite? Before, this section just linked to the upstream docs which mention Helm as a prerequisite. Helmfile is a separate tool right?